receiver/sqlserverreceiver/README.md

```diff
@@ -47,6 +47,14 @@ Top-Query collection specific options (only useful when top-query collection are
   - Queries that were finished execution outside the lookback window are not included in the collection. Increasing the lookback window (in seconds) will be useful for capturing long-running queries.
 - `max_query_sample_count` (optional, example = `5000`, default = `1000`): The maximum number of records to fetch in a single run.
 - `top_query_count`: (optional, example = `100`, default = `200`): The maximum number of active queries to report (to the next consumer) in a single run.
+- `collection_interval`: (optional, default = `60s`): The interval at which top queries should be emitted by this receiver.
+  - This value must be a string readable by Golang's [time.ParseDuration](https://pkg.go.dev/time#ParseDuration).
+  - This value can only guarantee that the top queries are collected at most once in this interval.
+    - For instance, you have global `collection_interval` as `10s` and `top_query_collection.collection_interval` as `60s`.
+      - In this case, the default receiver scraper will still try to run in every 10 seconds.
+      - However, the top queries collection will only run after 60 seconds have passed since the last collection.
+    - For instance, you have global `collection_interval` as `10s` and `top_query_collection.collection_interval` as `5s`.
+      - In this case, `top_query_collection.collection_internal` will make no effects to the collection
```

Comment:
This feels like it should be moved out from the attribute definitions, and into its own section that can explain it in more detail.

Reply:
This document is specifically intended for configurations, so it seems appropriate to include it here?